src/Schema/AST/ASTHelper.php

```diff
@@ -302,4 +303,36 @@ public static function typeImplementsInterface(ObjectTypeDefinitionNode $type, s
     {
         return self::firstByName($type->interfaces, $interfaceName) !== null;
     }
+
+    /**
+     * @param  \GraphQL\Language\AST\ObjectTypeDefinitionNode|\GraphQL\Language\AST\ObjectTypeExtensionNode  $objectType
+     * @param  \GraphQL\Language\AST\DirectiveNode  $directiveNode
+     * @return void
+     *
+     * @throws \Nuwave\Lighthouse\Exceptions\DefinitionException
+     */
+    public static function addDirectiveToFields(DirectiveNode $directiveNode, &$objectType): void
+    {
+        $name = $directiveNode->name->value;
+
+        if (
+            ! $objectType instanceof ObjectTypeDefinitionNode
+            && ! $objectType instanceof ObjectTypeExtensionNode
+        ) {
+            throw new DefinitionException(
+                "The @{$name} directive may only be placed on fields or object types."
+            );
+        }
+
+        /** @var \GraphQL\Language\AST\FieldDefinitionNode $fieldDefinition */
+        foreach ($objectType->fields as $fieldDefinition) {
+            // If the field already has the same directive defined, skip over it.
+            // Field directive are more specific then those defined on a type.
```

Comment:
shouldn't it be "Field **directives** are more specific **than** those defined on a type."?